generators/client/templates/angular/webpack/webpack.common.js.ejs

```diff
@@ -50,11 +50,12 @@ module.exports = (options) => ({
                 test: /\.html$/,
                 loader: 'html-loader',
                 options: {
-                    minimize: true,
-                    caseSensitive: true,
-                    removeAttributeQuotes:false,
-                    minifyJS:false,
-                    minifyCSS:false
+                    minimize: {
+                        caseSensitive: true,
+                        removeAttributeQuotes:false,
+                        minifyJS:false,
+                        minifyCSS:false
+                    }
```

Comment:
This change might be needed for React as well, to solve https://github.com/jhipster/generator-jhipster/issues/11651.

Reply:
Not sure, since `html-loader` plugin isn't used in the react side; since there are no HTML files I guess for React. :thinking: